x-pack/plugins/transform/common/constants.ts

```diff
@@ -190,6 +190,17 @@ export const TRANSFORM_HEALTH_CHECK_NAMES: Record<
       }
     ),
   },
+  healthCheck: {
+    name: i18n.translate('xpack.transform.alertTypes.transformHealth.healthCheckName', {
+      defaultMessage: 'Unhealthy transform',
+    }),
+    description: i18n.translate(
+      'xpack.transform.alertTypes.transformHealth.healthCheckDescription',
+      {
+        defaultMessage: 'Get alerts if a transform health status is not green.',
```

Comment:
I was wondering if this should be consistent with the messaging in the transforms list, using `Healthy` in place of `green`, but `Get alerts if a transform health status is not healthy.` reads a bit odd.